src/containers/SubjectPage/SubjectContainer.jsx

```diff
@@ -321,12 +293,20 @@ const SubjectPage = ({
 SubjectPage.getDocumentTitle = getDocumentTitle;
 
 SubjectPage.willTrackPageView = (trackPageView, currentProps) => {
-  const { data } = currentProps;
-  if (data?.subject?.topics?.length > 0) {
+  const { data, loading } = currentProps;
+  if (!loading && data?.subject?.topics?.length > 0) {
```

Comment:
Denne bør kun trigge når det ikkje er topics på sida. Altså kun for subject.
```suggestion
  if (!loading && data?.subject?.topics?.length === 0) {
```